core/web/user_controller.go

```diff
@@ -36,9 +220,14 @@ func (c *UserController) UpdatePassword(ctx *gin.Context) {
 		return
 	}
 
-	user, err := c.App.SessionORM().FindUser()
+	sessionUser, ok := webauth.GetAuthenticatedUser(ctx)
+	if !ok {
+		jsonAPIError(ctx, http.StatusInternalServerError, pkgerrors.Errorf("failed to obtain current user from context"))
+		return
+	}
+	user, err := c.App.SessionORM().FindUser(sessionUser.Email)
 	if err != nil {
-		jsonAPIError(ctx, http.StatusInternalServerError, fmt.Errorf("failed to obtain current user record: %+v", err))
+		jsonAPIError(ctx, http.StatusInternalServerError, pkgerrors.Errorf("failed to obtain current user record: %+v", err))
```

Comment:
It is usually OK to import as just `errors` and to drop the std lib import.

Reply:
Updated

Reply:
Bump. And I think you would prefer to use `errors.Wrapf(err, "...")` - `%+v` is quite odd here.

Reply:
Thank you for double checking, updated here and other instances and filed ticket 🙂